sess.go

```diff
@@ -32,13 +33,13 @@ type session struct {
 func (s *session) Serve() error {
 	t := protocol.NewTransport(s.Conn, s.Conn)
 
-	msg, err := t.StartUp()
+	suMsg, err := t.StartUp()
```

Comment:
instead of renaming to distinguish between startups messages and "pot-init" messages, what do you think about extracting to function similar to [what we did in pgproxy](https://github.com/panoplyio/pgproxy/blob/master/network/session.go#L60), and leave only main loop here?

Reply:
I did so many changes in the previous PR that I wanted to minimize unrequired changes. But if we're talking about it, I probably needed not to minimize :)

Reply:
hhh agree :)
splitting function to smaller functions is ok, diff is hugh, but easy to review:)